docs/v2/guides/terms.md

```diff
@@ -90,16 +92,14 @@ You can use this function in a similar way to how you use [`WP_Term_Query`](http
 ```php
 // Using the WP_Term_Query argument format.
 $term_query = new Timber::get_terms( [
-    'query' => [
-        'taxonomy' => 'book_genre',
-        'count'    => true,
-    ],
- ] );
+    'taxonomy' => 'book_genre',
+    'count'    => true,
+] );
 ```
 
-Be aware that you need to pass an array to `Timber::get_terms()` with a `query` parameter. This array can have additional arguments that you can check out in the documentation for [`Timber::get_terms()`](https://timber.github.io/docs/v2/reference/timber/#get-terms).
+Also check out the documentation for [`Timber::get_terms()`](https://timber.github.io/docs/v2/reference/timber/#get-terms).
 
-What you get as a return value is not a pure array of posts, but a `Timber\TermCollection` object, which is an `ArrayObject` that is very similar to an array as you know it. To loop over the terms collection in PHP, you first need to convert it to an array with `$terms->get_terms()`.
+What you get as a return value is not a pure array of term, but a `Timber\TermCollection` object, which is an `ArrayObject` that is very similar to an array as you know it. To loop over the terms collection in PHP, you first need to convert it to an array with `$terms->get_terms()`.
```

Comment:
This whole paragraph is not valid anymore and I replaced it with what’s actually being done, reflecting consideration 1).

> 1. In the documentation for [Querying Terms](https://timber.github.io/docs/v2/guides/terms/#querying-terms), I once wrote that there will be `TermCollections` that work similar to `PostCollections`. I guess I’ll have to remove that part, because we don’t have term collections yet?